src/librustc/util/ppaux.rs

```diff
@@ -540,6 +545,12 @@ impl fmt::Display for ty::Region {
             ty::ReSkolemized(_, br) => {
                 write!(f, "{}", br)
             }
+            ty::ReScope(code_extent) if identify_regions() => {
+                write!(f, "'{}ce", code_extent.index())
```

Comment:
Yes, `-Zverbose` is far too verbose, especially when it is embedded within a `&<region> <lvalue>` borrow-expression.

I don't want or care about the internal structure of the region value for these cases; it is much more convenient to look at the unique id's being generated here (by taking advantage of the fact that one cannot write a lifetime of the form `'<number><alphanum>`